rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -77,12 +78,28 @@ def pipeline_init(self, language):
             except ValueError as e:     # pragma: no cover
                 raise Exception("Duckling error. {}".format(e))
 
-    def process(self, text, entities):
+    def process(self, text, entities, ref_time=""):
         # type: (Text, List[Dict[Text, Any]]) -> Dict[Text, Any]
 
         extracted = []
         if self.duckling is not None:
-            matches = self.duckling.parse(text)
+            # check if ref_time is a datetime is in a known format
+            # If its not parseable, make it empty
+            if not ref_time:
+                ref_time = ""
```

Comment:
Right, but we might use that time for other things in the future (e.g. for logging)